Feature/homescreen

Sinbew opened the pull request.

Sinbew committed:
+ create-user
+ create-nickname and save it to async-storage
+ select device and save it to async storage

Sinbew committed:
add ChooseCharacterSheet

denysenkoa committed:
refactoring

Sinbew committed:
add ChooseCharacterSheet

denysenkoa committed:
refactoring

Sinbew committed:
Login screen:
Create user nickname sheet
Choose character sheet
Choose avatar sheet
Navigation to home screen

Sinbew committed:
Merge remote-tracking branch 'origin/develop' into feature/homescreen

# Conflicts:
#	app/constants/async-storage-key.ts
#	app/context/settings-context/SettingsProvider.tsx
#	app/screens/HomeScreen/HomeScreen.tsx
#	app/screens/LoginScreen/LoginScreen.tsx
#	app/screens/LoginScreen/sheets/ChooseAvatarSheet/ChooseAvatarSheet.tsx
#	app/screens/LoginScreen/sheets/ChooseCharacterSheet/ChooseCharacterSheet.tsx
#	app/screens/LoginScreen/sheets/ChooseCharacterSheet/views/CharacterCardView.tsx
#	app/screens/LoginScreen/sheets/CreateUserSheet/CreateUserSheet.tsx
#	app/screens/LoginScreen/sheets/CreateUserSheet/views/DevicesCardView.tsx

Sinbew committed:
Home-screen:
- Added statusbar
- Added views of:
 - nickname
 - level
 - points
 - team
 - progressbar
-Added CardViews:
 - Settings
 -About

denysenkoa commented on app/components/icons/WalknigIcon.tsx:
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```
typo
WalkingIcon

denysenkoa commented on app/screens/HomeScreen/views/ProgressBarView/ProgressBar.tsx:
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
ProgressBar -> ProgressBarView

Sinbew committed:
Home-screen:
- Added statusbar
- Added views of:
 - nickname
 - level
 - points
 - team
 - progressbar
-Added CardViews:
 - Settings
 -About
